UNRELEASED.md

```diff
@@ -2,6 +2,10 @@
 
 > Place your changes below this line.
 
+**Fixed:**
+- react-native-bpk-component-button
+  - `iconOnly` buttons no longer apply `boarderRadius` from theming. The iconOnly buttons now remain circular regardless of radius.
```

Comment:
I think this can be more elegantly worded for consumers, how about:

```suggestion
  - `iconOnly` buttons no longer apply `borderRadius` from theming and are always circular.
```